docs/recipes/install/common/confluent_init_os_images_rocky.tex

```diff
@@ -0,0 +1,44 @@
+% -*- mode: latex; fill-column: 120; -*-
+
+With the provisioning services enabled, the next step is to define
+a system image that can subsequently be
+used to provision one or more {\em compute} nodes. The following subsections highlight this process.
+
+\subsubsection{Build initial BOS image} \label{sec:assemble_bos}
+The following steps illustrate the process to build a minimal, default image for use with \Confluent{}. To begin, you will
+first need to have a local copy of the ISO image available for the underlying OS. In this recipe, the relevant ISO image
+is \texttt{Rocky-9.4-x86\_64-dvd1.iso} (available from the Rocky
```

Comment:
The image I downloaded does not have a "1" in the file name. The filename should be a variable so that it can be easily updated.